docs/plugins/spectate.rst

```diff
@@ -115,6 +127,15 @@ Settings
     If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
     happiness level (stress) of the dwarf in the tooltip.
 
+``tooltip-follow-stress-levels`` (default: Displeased, Content, Pleased are disabled)
+    If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
+    specific happiness level (stress) of the dwarf in the tooltip. F.e.
+    ``tooltip-follow-stress-levels 2 true`` would show Displeased emoticon.
```

Comment:
I think this needs a little more explanation. The full table of number - stress level name - default mappings would be useful, maybe in a separate doc section below since it's too big for adding here inline